src/core/pratter.ml

```diff
@@ -0,0 +1,159 @@
+(** This modules defines a functor whose image is a parser for terms with
+    applications, binary and unary operators. These terms are specified in the
+    argument of the functor.
+
+    The algorithm implemented is an extension of the Pratt parser. The
+    Shunting Yard algorithm could also be used.
+    @see <https://matklad.github.io/2020/04/13/simple-but-powerful-\
+pratt-parsing.html>
```

Comment:
Thanks to remove the \ and put the URL on one line so that we can easily copy and paste it. make sanity will not complain in this case. ;-)

Reply:
Anyway that file will eventually go away since it's part of the pratter library (and the line is not cut in the library)